uid-pages/page-admin-profile-list/src/adminProfileListBonitaV1.json

```diff
@@ -11434,7 +11434,7 @@
                                                 },
                                                 "label" : {
                                                   "type" : "interpolation",
-                                                  "value" : "{{$item.displayName}} {{editMembershipsMappingCtrl.isParentEmpty($item)}}"
+                                                  "value" : "{{editMembershipsMappingCtrl.getGroupDisplayName($item)}} {{editMembershipsMappingCtrl.isParentEmpty($item)}}"
```

Comment:
Why is this done? A group can never have no display name, it's always set to the value of name by the engine.

Reply:
In case, if the display name is different than the name. For example, 
name = hr, display name = Human Resources 
name = latin_america, display name = Latin America
This is what we currently have in the portal.